### PR TITLE
include format targeting flag implemented

### PR DESCRIPTION
**Pull request by BraslavskiyAndrey**

### CODE REVIEW

**Review by DGarbar — APPROVED** [review]

File: src/main/java/org/prebid/server/proto/openrtb/ext/response/BidType.java [hunk 1]
```diff
@@ -8,5 +8,12 @@
     video,
     audio,
     @JsonProperty("native")
-    xNative
+    xNative;
+
+    public String getName() {
+        if (this == xNative) {
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/proto/openrtb/ext/response/BidType.java:14]: Can be transformed to ternary operation.

File: src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java [hunk 2]
```diff
@@ -308,6 +309,55 @@ public void shouldReturnBidRequestWithIncludeWinnersFromStoredBidRequest() {
...
+
+        // when
+        final BidRequest request = factory.fromRequest(routingContext, 0L).result().getBidRequest();
+
+        // then
+        assertThat(singletonList(request))
+                .extracting(BidRequest::getExt).isNotNull()
+                .extracting(ExtRequest::getPrebid)
+                .extracting(ExtRequestPrebid::getTargeting)
+                .extracting(ExtRequestTargeting::getIncludeformat, ExtRequestTargeting::getPricegranularity)
```
Comment (rpanchyk) [inline src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java:331]: Do we need to test price granularity here?

File: src/test/resources/org/prebid/server/it/amp/test-appnexus-bid-request.json [hunk 3]
```diff
@@ -81,7 +81,8 @@
           ]
         },
         "includewinners": true,
-        "includebidderkeys": true
+        "includebidderkeys": true,
+        "includeformat" : false
```
Comment (rpanchyk) [inline src/test/resources/org/prebid/server/it/amp/test-appnexus-bid-request.json:85]: Do we want to add defaulting value in bidder request if incoming `includeformat` flag is not defined?

**Review by rpanchyk — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java [hunk 4]
```diff
@@ -247,6 +266,9 @@ private static PriceGranularity convertToCustomPriceGranularity(String stringPri
             keywordMap.put(HB_CACHE_HOST_KEY, cacheHost);
             keywordMap.put(HB_CACHE_PATH_KEY, cachePath);
         }
+        if (StringUtils.isNotBlank(format) && BooleanUtils.toBooleanDefaultIfNull(includeFormat, false)) {
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/auction/TargetingKeywordsCreator.java:269]: Can we use `BooleanUtils.isTrue(..)` instead?

**Review by rpanchyk — CHANGES_REQUESTED** [review]

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**